packages/design-systems/src/components/composite/Accordion.tsx

```diff
@@ -0,0 +1,54 @@
+import { Accordion as ArkAccordion } from "@ark-ui/react";
+import { type HTMLStyledProps } from "@tailor-platform/styled-system/jsx";
+import { accordion } from "@tailor-platform/styled-system/recipes";
+import { ChevronDown } from "lucide-react";
+import { Button } from "../Button";
+import { Text } from "../Text";
+
+export type AccordionItem = {
+  text: string;
+  content: string;
```

Comment:
contentには何かしらのコンポーネントを表示するケースもありますので、ここはReactNodeでお願いしたいです
```suggestion
  content: React.ReactNode;
```